packages/presentation/lib/ui/splash_screen/splash_widget.dart

```diff
@@ -23,11 +27,62 @@ class SplashWidget extends StatefulWidget {
 }
 
 class _SplashWidgetState extends BlocScreenState<SplashWidget, SplashBloc> {
+  Future<void> showMyDialog(
```

Comment:
create separate stream, which we will use for showing dialogs. I think we did smth similar in guess number

Reply:
created a separate stream, in case of ui events + got rid of the basic stream